netlify.toml

```
@@ -319,7 +319,7 @@
 
 [[redirects]]
   from = "/guides/dashboard/analytics"
-  to = "/guides/cloud/analytics"
+  to = "/guides/cloud/analytics/overview"
```

we should add `guides/cloud/analytics` to this to avoid unexpected 404s

```suggestion
  to = "/guides/cloud/analytics/overview"
  status = 301
  force = true
  
 [[redirects]]
  from = "/guides/cloud/analytics"
  to = "/guides/cloud/analytics/overview"
```